README.md

```diff
@@ -83,10 +83,9 @@ To learn deeply about KubeEdge, try some examples on [examples](https://github.c
 ## Meeting
 
 Regular Community Meeting:
-- Europe Time: **Wednesdays at 16:30-17:30 Beijing Time** (biweekly, starting from Feb. 19th 2020).
+- Europe Time: **Wednesdays at 16:00-17:30 Beijing Time** (weekly, starting from Feb. 19th 2020).
 ([Convert to your timezone.](https://www.thetimezoneconverter.com/?t=16%3A30&tz=GMT%2B8&))
-- Pacific Time: **Wednesdays at 10:00-11:00 Beijing Time** (biweekly, starting from Feb. 26th 2020).
-([Convert to your timezone.](https://www.thetimezoneconverter.com/?t=10%3A00&tz=GMT%2B8&))
```

Comment:
thanks for the update! this means the biweekly meeting in pacific time is shut down and keep a dedicated weekly meeting in Europe time, right? (it is not really friendly time for US, but i wanna join some time... 😅 )

Reply:
The operation has been canceled; I just want to correct the original incorrect Europe Time.